lib/reports/ReportPedDelaySummary.js

```diff
@@ -0,0 +1,455 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
we should _eventually_ add tests here (e.g. CSV generation, number of pages in PDF, end-to-end comparison with legacy reports) - but again, this might be better consigned to a separate PR.

(we'll have to create a test dataset for ped delay counts, and transcribe an old report to compare against - both of which are non-trivial.)

Reply:
added to #972 .